oe_theme.theme

```diff
@@ -955,28 +955,14 @@ function oe_theme_preprocess_form__oe_search_search_form(array &$variables): voi
  *   The ECL array for the input element.
  */
 function _oe_theme_preprocess_search_input_text(array $element): array {
-  // Process element attributes into an ecl input array.
+  // Process element attributes into an ECL input array.
   $ecl_array = [
     'id' => $element['#attributes']['id'],
     'name' => $element['#attributes']['name'],
     'disabled' => isset($element['#attributes']['disabled']) ? $element['#attributes']['disabled'] : FALSE,
     'type' => $element['#attributes']['type'],
     'class' => $element['#attributes']['class'],
   ];
-  $ecl_array['class'][] = 'ecl-search-form__text-input';
-
-  $extra_attributes = !empty($element['#attributes']) ? $element['#attributes'] : [];
-  $extra_attributes = array_diff_key($extra_attributes, $ecl_array);
-
-  $ecl_array['extra_attributes'] = [];
-  foreach ($extra_attributes as $key => $value) {
```

Comment:
We are removing the extra_attributes from the text_input here, is this intentional?

Reply:
Apparently the extra_attributes on the text_input have no effect so removing them doesn't affect anything.